### PR TITLE
dietary info

### PR DESCRIPTION
**Pull request by bluesheepllama**
added all the dietary info

### CODE REVIEW

File: shared/data/DatabaseItems.csv [hunk 1]
```diff
@@ -1,167 +1,167 @@
...
+Lighthouse Cafe,Grilled Chicken Breast Sandwich,6.99,"vitalian marinated chicken breast, lettece, onion, tomato, pickles, ranch dressing",grilled_chicken_breast_sandwich.jpg
+Lighthouse Cafe,Grilled Cheese Sandwhich,4.29,"provolone, american cheese, butter. Vegetarian.",grilled_cheese.jpg
+Lighthouse Cafe,Buffalo Chicken Wrap,7.99,"crispy chicken, chedder jack cheese, lettuce,tomato, buffalo sauce in a flour tortilla",chicken_wrap.jpg
+Lighthouse Cafe,Cripsy Chicken Buffalo Wrap,8.69,"crispy chicken, bacon, chedder jack cheese, lettuce, ranch in a flour tortilla. Low Carb.",chicken_wrap.jpg
+Lighthouse Cafe,Crispy Chicken Strips,6.99,3 battered and fried chicken tenders served with choice of dipping sauce,chicken_strips.jpg
+Lighthouse Cafe,Veggie Wrap,5.59,"Spring mix. shredded carrots, tomatoes, cucumber, avacado, balsamic dressing in a flour tortilla. Vegetarian.. Vegan. Dairy Free.",veggie_wrap.jpg
+Lighthouse Cafe,Salad Of The Week,3.99,changes every week. Vegetarian. Vegan. Dairy Free. Low Carb.,salad.jpg
+Lighthouse Cafe,Soup Of The Week,3.49,changes every week,soup.jpg
+Lighthouse Cafe,Fries,2.59,choice of sauce Vegetarian. Vegan. Dairy Free.,fries.jpg
```
Comment (kelseyp) [inline shared/data/DatabaseItems.csv]:
```suggestion
Lighthouse Cafe,Fries,2.59,choice of sauce. Vegetarian. Vegan. Dairy Free.,fries.jpg
```

File: shared/data/DatabaseItems.csv [hunk 2]
```diff
@@ -1,167 +1,167 @@
...
+Lighthouse Cafe,Turkey Burger,6.39,"provolone, lettece, onion, tomato, 1000 island sauce",turkey_burger.jpg
+Lighthouse Cafe,Veggie or Black Bean Burger,6.39,"veggie or black bean patti, american cheese, lettece, onion, tomato, 1000 island sauce. Vegetarian.",veggie_burger.jpg
+Lighthouse Cafe,Grilled Chicken Breast Sandwich,6.99,"vitalian marinated chicken breast, lettece, onion, tomato, pickles, ranch dressing",grilled_chicken_breast_sandwich.jpg
+Lighthouse Cafe,Grilled Cheese Sandwhich,4.29,"provolone, american cheese, butter. Vegetarian.",grilled_cheese.jpg
+Lighthouse Cafe,Buffalo Chicken Wrap,7.99,"crispy chicken, chedder jack cheese, lettuce,tomato, buffalo sauce in a flour tortilla",chicken_wrap.jpg
+Lighthouse Cafe,Cripsy Chicken Buffalo Wrap,8.69,"crispy chicken, bacon, chedder jack cheese, lettuce, ranch in a flour tortilla. Low Carb.",chicken_wrap.jpg
+Lighthouse Cafe,Crispy Chicken Strips,6.99,3 battered and fried chicken tenders served with choice of dipping sauce,chicken_strips.jpg
+Lighthouse Cafe,Veggie Wrap,5.59,"Spring mix. shredded carrots, tomatoes, cucumber, avacado, balsamic dressing in a flour tortilla. Vegetarian.. Vegan. Dairy Free.",veggie_wrap.jpg
```
Comment (kelseyp) [inline shared/data/DatabaseItems.csv]:
```suggestion
Lighthouse Cafe,Veggie Wrap,5.59,"Spring mix. shredded carrots, tomatoes, cucumber, avacado, balsamic dressing in a flour tortilla. Vegetarian. Vegan. Dairy Free.",veggie_wrap.jpg
```

File: shared/data/DatabaseItems.csv [hunk 3]
```diff
@@ -1,167 +1,167 @@
...
+Mom Wong Kitchen,Green Onion Pancake,6.99,Pancake made with green onion. Vegetarian.,Green_Onion_Pancake.jpg
+Mom Wong Kitchen,Pork Pot Sticker,10.49,Dumpling filled with pork. Dairy Free.,Pork_Pot_Sticker.jpg
+Mom Wong Kitchen,Red Bean Bun,3.99,Bun made with Red Bean. Vegetarian.,Red_Bean_Bun.jpg
+Mom Wong Kitchen,Egg Roll,1.99,Egg and vegetables. Vegetarian. Dairy Free.,Egg_roll.jpg
+Mom Wong Kitchen,Beef Pancake Roll,10.49,Beef rolled in a pancake,Beef_Pancake_Roll.jpg
+Mom Wong Kitchen,Chicken Dumpling,10.49,"Dumpling with chicken on the inside, can be boiled, fried or steamed. Dairy Free.",Chicken_Dumpling.jpg
+Mom Wong Kitchen,Beef Pot Sticker,10.49,Dumpling filled with beef. Dairy Free.,Beef_Pot_Sticker.jpg
+Mom Wong Kitchen,Juicy Pork Dumpling w/ Peanuts,10.49,Dumpling filled with juicy pork and peanuts. Dairy Free.,Juicy_Pork_Dumpling_With_Peanuts.jpg
+Mom Wong Kitchen,Pork and Napa Dumpling,10.49,Dumpling filled with Pork and Napa. Dairy Free.,Pork_and_Napa_Dumpling.jpg
+Mom Wong Kitchen,Fried Cruller,1.49,Deep-fried pastry. Vegetarian. ,Fried_Cruller.jpg
```
Comment (kelseyp) [inline shared/data/DatabaseItems.csv]:
```suggestion
Mom Wong Kitchen,Fried Cruller,1.49,Deep-fried pastry. Vegetarian.,Fried_Cruller.jpg
```

File: shared/data/DatabaseItems.csv [hunk 4]
```diff
@@ -1,167 +1,167 @@
...
+Mom Wong Kitchen,Vegetable Dumpling,9.99,Dumpling filled with vegetables. Vegetarian. Vegan. Dairy Free.,Vegetable_Dumpling.jpg
+Mom Wong Kitchen,Garlic Flavored Fried Chicken,9.99,Fried chicken with garlic flavoring,Garlic_Flavored_Fried_Chicken.jpg
+Mom Wong Kitchen,Shrimp and Chive Egg Dumpling,10.49,"Dumpling filled with shrimp and chive egg, can be boiled, fried, or steamed. Dairy Free",Shrimp_and_Chive_Egg_Dumpling.jpg
```
Comment (kelseyp) [inline shared/data/DatabaseItems.csv]:
```suggestion
Mom Wong Kitchen,Shrimp and Chive Egg Dumpling,10.49,"Dumpling filled with shrimp and chive egg, can be boiled, fried, or steamed. Dairy Free.",Shrimp_and_Chive_Egg_Dumpling.jpg
```

File: shared/data/DatabaseItems.csv [hunk 5]
```diff
@@ -1,167 +1,167 @@
...
+Mom Wong Kitchen,Pork Hand-Ripped Noodle with Special Sauce,11.49,"Noodles made by hand-pulling wheat flour dough, served with pork and special sauce. Dairy Free",Pork_Hand-Ripped_Noodle_with_Special_Sauce.jpg
+Mom Wong Kitchen,Beef Hand-Ripped Noodle with Special Sauce,11.49,"Noodles made by hand-pulling wheat flour dough, served with beef and special sauce. Dairy Free",Beef_Hand-Ripped_Noodle_with_Special_Sauce.jpg
+Mom Wong Kitchen,Vegetables Hand-Ripped Noodle with Special Sauce,9.99,"Noodles made by hand-pulling wheat flour dough, served with vegetables and special sauce. Vegetarian. Vegan. Dairy Free.",Vegetables_Hand-Ripped_Noodle_with_Special_Sauce.jpg
+Mom Wong Kitchen,Chicken Hand-Ripped Noodle with Special Sauce,11.49,"Noodles made by hand-pulling wheat flour dough, served with chicken and special sauce. Dairy Free.",Chicken_Hand-Ripped_Noodle_with_Special_Sauce.jpg
+Mom Wong Kitchen,Shrimp Hand-Ripped Noodle with Special Sauce,11.49,"Noodles made by hand-pulling wheat flour dough, served with shrimp and special sauce Dairy Free.",Shrimp_Hand-Ripped_Noodle_with_Special_Sauce.jpg
```
Comment (kelseyp) [inline shared/data/DatabaseItems.csv]:
```suggestion
Mom Wong Kitchen,Shrimp Hand-Ripped Noodle with Special Sauce,11.49,"Noodles made by hand-pulling wheat flour dough, served with shrimp and special sauce. Dairy Free.",Shrimp_Hand-Ripped_Noodle_with_Special_Sauce.jpg
```

File: shared/data/DatabaseItems.csv [hunk 6]
```diff
@@ -1,167 +1,167 @@
...
+Mom Wong Kitchen,Cold Noodle with Sesame Sauce,9.99,Cold noodles served with a sesame sauce. Vegetarian. Vegan. Dairy Free.,Cold_Noodle_with_Sesame_Sauce.jpg
+Mom Wong Kitchen,Orange Chicken,11.49,Orange chicken served with rice.,Orange_Chicken.jpg
+Mom Wong Kitchen,Kung Pao Chicken,11.49,Kung Pao Chicken served with rice. Dairy Free.,Kung_Pao_Chicken.jpg
+Mom Wong Kitchen,Stir-Fried Green Beans,10.49,Stir-Fried Green Beans served with rice. Vegetarian. Vegan. Dairy Free. Low Carb.,Stir-Fried_Green_Beans.jpg
+Mom Wong Kitchen,Mapo Tofu,10.49,Tofu served with rice. Vegetarian. Vegan. Dairy Free.,Mapo_Tofu.jpg
+Mom Wong Kitchen,Popcorn Chicken,8.99,Popcorn chicken served with rice,Popcorn_Chicken.jpg
+Mom Wong Kitchen,Chongqing Style Fried Spicy Chicken,11.49,Chongqing Style Fried Spicy Chicken served with rice,Chongqing_Style_Fried_Spicy_Chicken.jpg
+Mom Wong Kitchen,Cucumber with Garlic Sauce,4.99,Cucumber with garlic sauce drizzled on. Vegetarian. Vegan. Dairy Free. Low Carb.,Cucumber_with_Garlic_Sauce.jpg
+Mom Wong Kitchen,Stir-Fried Baby Bok Choy,10.49,Stir-Fried Baby Bok Choy served with rice. Vegetarian.. Vegan. Dairy Free. Low Carb.,Stir-Fried_Baby_Bok_Choy.jpg
```
Comment (kelseyp) [inline shared/data/DatabaseItems.csv]:
```suggestion
Mom Wong Kitchen,Stir-Fried Baby Bok Choy,10.49,Stir-Fried Baby Bok Choy served with rice. Vegetarian. Vegan. Dairy Free. Low Carb.,Stir-Fried_Baby_Bok_Choy.jpg
```

**Review by kelseyp — CHANGES_REQUESTED** [review]
I've added a number of suggestions for correcting typos. Additionally, is it correct to remove most double quotes around descriptions and image names? How has this been tested?

File: shared/data/DatabaseItems.csv [hunk 7]
```diff
@@ -1,167 +1,167 @@
...
-Pizza 3.14,Mozzarella Sticks w/Marinara,4.49,Mozzarella Cheese Breaded and Fried,"Mozzarella_Sticks.jpg"
-Pizza 3.14,Small Salad,2.79,Lettuce and Vegetables,"salad.jpg"
-Pizza 3.14,Large Salad,5.79,Lettuce and Vegetables,"salad.jpg"
-Pizza 3.14,3.14 Zookie,3.79,NA,"zookie.jpg"
-Pizza 3.14,Albert's Brownie,3.79,"Chocolate, Milk, dessert","brownie.jpg"
+Pizza 3.14,Seasoned Potato Wedges w/Ranch,2.79,"Fired Potatoes. Vegetarian. Vegan. Dairy Free.","Potato_Wedges.jpg"
+Pizza 3.14,Onion Rings,3.99,"Fried Breaded Onions. Vegetarian.","onion_rings.jpg"
+Pizza 3.14,Sweet Potato Fries,2.5,"Fried Sweet Potatoes. Vegetarian. Vegan. Dairy Free.","Sweet_Potato_Fries.jpg"
+Pizza 3.14,Side of Chips,1.5,"Sliced Potatoes and Salt. Vegetarian. Vegan. Dairy Free.","chips.jpg"
+Pizza 3.14,Garlic Bread,3,"Bread with Garlic and Butter. Vegetarian.","garlic_bread.jpg"
```
Comment (kelseyp) [inline shared/data/DatabaseItems.csv:29]: Did you mean to remove the 0's from the end of these prices?

**Review by kelseyp — APPROVED** [review]

**Review by NathanaelPaulus — APPROVED** [review]
Good job! Can't wait to see it all come together

**Pull request merged**